docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -619,6 +624,17 @@ code: |
   interview_download.initialize(filename = interview.file_name + '.yml')
   interview_download.write(interview.source())
   interview_download.commit()
+
+  folders_and_files = {
+    "templates": template_upload,
+    "questions": [interview_download],
+    "modules":[],
+    "static": [],
```

Comment:
Not for right now, but would it be worth it to eventually include some template css files in here. Completely empty, but with some commented out options for changing the theme?